hack/triage/config.yaml

```diff
@@ -0,0 +1,269 @@
+      - label: "!triage/accepted"
+      - label: "priority/critical-urgent"
+      - label: "lifecycle/(stale|rotten)"
+      - commented: +30d
+
+  triage-lifecycle-stale-or-rotten-priority-important-soon:
+    # WHY?: issue with priority/important-soon  do not get automatically closed, it is up to maintainers to take a look again
+    # Note: this is a clear sign of project lacking contributors. At some point - might be after two back and forth -, we should give up and close.
+    name: "Issue with priority/important-soon, lifecycle/stale or rotten, not commented in the last 2 month"
+    resolution: "Consider if to change priority with /priority important-longterm or /priority backlog, consider if to close with /close"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+      - label: "priority/important-soon"
+      - label: "lifecycle/(stale|rotten)"
+      - commented: +60d
+
+  triage-lifecycle-stale-or-rotten-priority-important-longterm:
+    # WHY?: issue with priority/important-longterm do not get automatically closed, it is up to maintainers to take a look again
+    # Note: this is a clear sign of project lacking contributors. At some point - might be after two back and forth -, we should give up and close.
+    name: "Issue with priority/important-longterm, lifecycle/stale or rotten, not commented in the last 6 months"
+    resolution: "Consider if to change priority with /priority backlog, consider if to clos with /closee"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+      - label: "priority/important-longterm"
+      - label: "lifecycle/(stale|rotten)"
+      - commented: +180d
+
+  triage-lifecycle-frozen:
+    # WHY?: issue with lifecycle/frozen do not get automatically closed, it is up to maintainers to take a look again
+    # Note: this is a clear sign of project lacking contributors. At some point - might be after two back and forth -, we should give up and close.
+    name: "Issue with lifecycle/frozen, not commented in the last 6 months"
+    resolution: "Consider if to close with /close"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+      - label: "lifecycle/frozen"
+      - commented: +180d
+
+  triage-all:
+    name: "All the issue in triage"
+    resolution: "Check if the issue is now actionable by applying /triage accepted, help in making progress, consider if to change priority or kind"
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+
+  # Phase: Actionable
+
+  actionable-not-assigned-without-help-or-good-first-issue:
+    # WHY?: issue actionable must be assigned to someone or seeking for help
+    name: "Issues not assigned without help or good-first-issue"
+    resolution: "Apply /help or /good-first-issue"
+    type: issue
+    filters:
+      - label: "triage/accepted"
+      - label: "!(help|good-first-issue)"
+      - tag: "!assigned"
+
+  actionable-updated:
+    # WHY?: maintainers are supposed to keep up with issue discussions
```

Comment:
Fine to keep it, but we really have to be careful how we work with this list. We shouldn't try to get this one to 0 by always having to post something on every issue last. This could get very annoying quickly

Reply:
Added: "Note: Is is be okay if this list does not go to 0, maintainers are not forced to add an answer to get this list clean."